src/components/Profile.astro

```diff
@@ -0,0 +1,56 @@
+---
+import { Image } from 'astro:assets';
+import type { Author } from '../pages/themes/_types/index.ts';
+
+interface Props {
+	author: Author;
+}
+
+const { author } = Astro.props;
+---
+
+<div class="flex lg:flex-col flex-wrap items-center gap-4">
+	<Image
+		inferSize
+		src={author.avatar ?? ''}
+		height={48}
+		width={48}
+		class="rounded-full size-12 lg:w-full h-fit border"
+		alt="Your avatar"
+	/>
+	<h1 class="border-astro-gray-500 lg:hidden heading-3">{author.name}</h1>
+	<div class="flex flex-col gap-4 pt-2 lg:pt-8 w-full">
+		<div class="flex items-center justify-between">
+			<small class="code text-astro-gray-200">Joined</small>
+			<small class="text-astro-gray-200">
+			{
+				new Date(author.createdAt).toLocaleDateString('en-US', {
+					year: 'numeric',
+					month: 'long',
+					day: 'numeric',
+				})
+			}
+			</small>
+		</div>
+		<hr class="border-astro-gray-500" />
+		<div class="flex items-center justify-between">
+			<small class="code text-astro-gray-200">GitHub</small>
+			<small class="text-astro-gray-200">
+				<a class="link" href={`https://github.com/${author.username}`}>@{author.username}</a>
+			</small>
+		</div>
+		{
+			author.url && (
+				<hr class="border-astro-gray-500" />
+				<div class="flex items-center justify-between">
+					<small class="code text-astro-gray-200">Website</small>
+					<small class="text-astro-gray-200">
+						<a class="link" href={author.url}>
+							{author.url}
+						</a>
+					</small>
+				</div>
+			)
+		}
+	</div>
```

Comment:
I think the text size in this sidebar section may need tweaking. I’d use the right sidebar on theme pages as a reference.

For comparison:

| | |
|---|---|
| <img width="332" alt="image" src="https://github.com/user-attachments/assets/5c1129d5-3d54-4aa7-84b7-fb6ce6403659" /> | <img width="417" alt="image" src="https://github.com/user-attachments/assets/833e6750-9050-4316-9e7b-45a182f5f850" /> |

In the theme page sidebar, values are slightly larger.

Reply:
Removed the small tag, but the link styles will still make it slightly darker and give the on hover effect